app/controllers/api/order_details_controller.rb

```diff
@@ -3,7 +3,7 @@ class Api::OrderDetailsController < ApplicationController
   respond_to :json
   rescue_from ActiveRecord::RecordNotFound, with: :order_detail_not_found
 
-  http_basic_authenticate_with name: Settings.api.basic_auth_name, password: Settings.api.basic_auth_password
+  http_basic_authenticate_with name: Rails.application.secrets.api["basic_auth_name"], password: Rails.application.secrets.api["basic_auth_password"]
```

Comment:
Just a warning, in Rails 5.1, the keys become symbols. http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#application-secrets-now-loaded-with-all-keys-as-symbols

Reply:
Ah okay, will make sure to update that when we move to 5.1.